github/data_source_github_repository_branches.go

```diff
@@ -16,6 +16,16 @@ func dataSourceGithubRepositoryBranches() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"only_protected_branches": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
What do you think about adding a `ConflictsWith` for `only_protected_branches` and `only_non_protected_branches`? Currently, if both are set to true, only protected branches will be retrieved, which might be a little confusing.

Reply:
Done.